jabbercat/models.py

```diff
@@ -1,28 +1,27 @@
 import bisect
-import collections.abc
 import enum
 import typing
 import unicodedata
-
-import lxml.builder
-import lxml.etree
+from itertools import chain
```

Comment:
Just use ``import itertools`` and ``itertools.chain`` down the line.

Reply:
OK